overlord/configstate/configcore/log_control.go

```diff
@@ -0,0 +1,79 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
```

Comment:
We might want to support more (or all) /proc/sys params over time, not sure we want a single file for each of them. Like we have a single services.go, maybe this should be called sysctl.go.